docs/domains/Domain.md

```diff
@@ -157,6 +158,16 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | periodSeconds | number | The number of seconds between checks |
 | timeoutSeconds | number | The number of seconds with no response that indicates a failure |
 
+### Shutdown
+
+Shutdown describes the configuration for shutting down a server instance.
+
+| Name | Type | Description |
+| --- | --- | --- |
+| ignoreSessions | boolean | For graceful shutdown only, indicates to ignore pending HTTP sessions during in-flight work handling. Not required. Defaults to false. |
```

Comment:
ignoreSessions -> `ignoreSessions`
boolean -> Boolean
shutdownType -> `shutdownType`
timeoutSeconds -> `timeoutSeconds`

Reply:
I'll see if I can fix this... This is generated content :)

Reply:
Not a big deal.

Reply:
https://github.com/oracle/weblogic-kubernetes-operator/pull/1032 😄